packages/integration-tests/api/distribution.js

```diff
@@ -106,30 +106,22 @@ async function invokeTEADistributionLambda(
 }
 
 /**
- * Invoke s3-credentials-endpoint lambda directly to get s3 credentials. This
- * is used in integration testing so that we use the lambda's IAM
- * role/permissions when accessing resources.
+ * Invoke s3-credentials-endpoint lambda directly to get s3 credentials.=
```

Comment:
Was this change an accident?

Reply:
No.  I feel the sentence `This is used in..` is redundant